### PR TITLE
New issue template for CRM updates

### PR DESCRIPTION
**Pull request by AnthonyRollins**

### CODE REVIEW

**Comment by AnthonyRollins** [comment]
@thekaveman, Let me know if this looks good, but also if there was anything I should have done differently about creating a PR.

File: .github/ISSUE_TEMPLATE/crm-record-update.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: "CRM Record Update"
+description: Update a record inside Hubspot
+title: "[Deals Pipeline Request]: "
```
Comment (thekaveman) [inline .github/ISSUE_TEMPLATE/crm-record-update.yml:3]: I think you'll want to update the `title` here

File: .github/ISSUE_TEMPLATE/crm-record-update.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+labels:
+  - "crm-update"
+body:
+  - type: markdown
+    attributes:
+      value: |
+       ### Due to sensitive information, please submit your crm record request through Hubspot's Service Desk pipeline; [Internal Support](https://app.hubspot.com/contacts/5519226/objects/0-5/views/all/board). You won't need to submit this issue, as you should submit this request through Hubspot.
```
Comment (thekaveman) [inline .github/ISSUE_TEMPLATE/crm-record-update.yml:10]: Nice! 👍

**Review by thekaveman — CHANGES_REQUESTED** [review]
Your PR looks good! I saw one minor thing to change

**Comment by AnthonyRollins** [comment]
Made all changes

**Review by thekaveman — APPROVED** [review]
Looks great!

Just be sure to create the `crm-update` label before anyone uses this template.

**Pull request merged**